### PR TITLE
Added new phishing campaign dynamic section scripts

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/40396

## Description
Added new campaign showing scripts for dynamic sections in phishing campaign layout.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Campaign/Scripts/PerformActionOnCampaignIncidents/PerformActionOnCampaignIncidents.py [hunk 1]
```diff
@@ -120,8 +124,15 @@ def perform_take_ownership(ids, action):
 
 def main():
     try:
-        action = get_custom_field(ACTION_ON_CAMPAIGN_FIELD_NAME).lower()
+        action = get_custom_field(ACTION_ON_CAMPAIGN_FIELD_NAME)
```
Comment (dantavori) [inline Packs/Campaign/Scripts/PerformActionOnCampaignIncidents/PerformActionOnCampaignIncidents.py:127]: as we spoke - use an argument to determine the similarity type (high/low)

File: Packs/Campaign/Scripts/GetCampaignLowerSimilarityIncidentsIdsAsOptions/GetCampaignLowerSimilarityIncidentsIdsAsOptions.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+commonfields:
+  id: GetCampaignLowerSimilarityIncidentsIdsAsOptions
+  version: -1
+name: GetCampaignLowerSimilarityIncidentsIdsAsOptions
+script: ''
+type: python
+tags:
+- field-display
+comment: Get the campaign lower similarity incident IDs as option values for Multi
+  Select field.
```
Comment (dantavori) [inline Packs/Campaign/Scripts/GetCampaignLowerSimilarityIncidentsIdsAsOptions/GetCampaignLowerSimilarityIncidentsIdsAsOptions.yml:10]:
```suggestion
comment: Gets the IDs of incidents with lower similarity. Used to fill the optional values of the multi-select "Phishing Campaign Select Campaign Lower Similarity Incidents" incident field.
```

File: Packs/Campaign/Scripts/GetCampaignLowerSimilarityIncidentsIdsAsOptions/GetCampaignLowerSimilarityIncidentsIdsAsOptions_test.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+def test_there_is_no_id_in_incident(mocker):
+    """
+    Given -
+        Incident in campaign in context doesn't have id
+
+    When -
+        Try to get the ids for the multi select field
+
+    Then -
+        Validate the
```
Comment (dantavori) [inline Packs/Campaign/Scripts/GetCampaignLowerSimilarityIncidentsIdsAsOptions/GetCampaignLowerSimilarityIncidentsIdsAsOptions_test.py:83]: please complete the docstring

File: Packs/Campaign/Scripts/GetCampaignLowerSimilarityIncidentsIdsAsOptions/GetCampaignLowerSimilarityIncidentsIdsAsOptions_test.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+    # prepare
+    mocker.patch.object(demisto, 'results')
+    no_ids_incidents = [{'name': f'test_{i}' for i in range(NUM_OF_INCIDENTS)}]
+    mocker.patch('GetCampaignLowerSimilarityIncidentsIdsAsOptions.get_campaign_incidents', return_value=no_ids_incidents)
+
+    # run
+    try:
+        GetCampaignLowerSimilarityIncidentsIdsAsOptions.main()
+    except SystemExit:
```
Comment (dantavori) [inline Packs/Campaign/Scripts/GetCampaignLowerSimilarityIncidentsIdsAsOptions/GetCampaignLowerSimilarityIncidentsIdsAsOptions_test.py:95]:
```suggestion
    try:
        GetCampaignLowerSimilarityIncidentsIdsAsOptions.main()
        assert False
    except SystemExit:
```

File: Packs/Campaign/Scripts/ShowCampaignLastIncidentOccurred/ShowCampaignLastIncidentOccurred.py [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+        if last_incident_occurred:
+            html_readable_output = f"<div style='text-align:center; font-size:17px; padding: 15px;'>" \
+                                   f"Last Incident Occurred</br> <div style='font-size:24px;'> " \
+                                   f"{last_incident_occurred} </div></div>"
+
+        else:
+            html_readable_output = "<div style='text-align:center; font-size:17px; padding: 15px;'>" \
+                                   "Last Incident Occurred</br> <div style='font-size:20px;'> " \
+                                   "No last incident occurred found. </div></div>"
```
Comment (dantavori) [inline Packs/Campaign/Scripts/ShowCampaignLastIncidentOccurred/ShowCampaignLastIncidentOccurred.py:59]:
```suggestion
                                   "N/A</div></div>"
```

File: Packs/Campaign/Scripts/ShowCampaignLastIncidentOccurred/ShowCampaignLastIncidentOccurred.yml [hunk 6]
```diff
@@ -0,0 +1,18 @@
+commonfields:
+  id: ShowCampaignLastIncidentOccurred
+  version: -1
+name: ShowCampaignLastIncidentOccurred
+script: ''
+type: python
+tags:
+- dynamic-section
+comment: Displays the date of the last campaign incident occurred.
```
Comment (dantavori) [inline Packs/Campaign/Scripts/ShowCampaignLastIncidentOccurred/ShowCampaignLastIncidentOccurred.yml:9]:
```suggestion
comment: Displays the occurrence date of the last campaign incident.
```

File: Packs/Campaign/Scripts/ShowCampaignRecipients/ShowCampaignRecipients.yml [hunk 7]
```diff
@@ -0,0 +1,18 @@
+commonfields:
+  id: ShowCampaignRecipients
+  version: -1
+name: ShowCampaignRecipients
+script: ''
+type: python
+tags:
+- dynamic-section
+comment: Displays the phishing campaign recipients email addreses and their amount.
```
Comment (dantavori) [inline Packs/Campaign/Scripts/ShowCampaignRecipients/ShowCampaignRecipients.yml:9]:
```suggestion
comment: Displays the phishing campaign recipients email addresses and their amount.
```

File: Packs/Campaign/Scripts/ShowCampaignSenders/ShowCampaignSenders.yml [hunk 8]
```diff
@@ -0,0 +1,18 @@
+commonfields:
+  id: ShowCampaignSenders
+  version: -1
+name: ShowCampaignSenders
+script: ''
+type: python
+tags:
+- dynamic-section
+comment: Displays the phishing campaign senders email addreses and their amount.
```
Comment (dantavori) [inline Packs/Campaign/Scripts/ShowCampaignSenders/ShowCampaignSenders.yml:9]:
```suggestion
comment: Displays the phishing campaign senders email addresses and their amount.
```

File: Packs/Campaign/ReleaseNotes/3_0_2.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+- Added support for campaign lower similarity incident fields.
+
+##### New: ShowCampaignSimilarityRange
+- Displays the similarity range between the incidents that make up the phishing campaign. (Available from Cortex XSOAR 5.5.0).
+
+##### New: ShowCampaignRecipients
+- Displays the phishing campaign recipients email addreses and their amount. (Available from Cortex XSOAR 5.5.0).
+
+##### New: ShowCampaignLastIncidentOccurred
+- Displays the date of the last campaign incident occurred. (Available from Cortex XSOAR 5.5.0).
```
Comment (dantavori) [inline Packs/Campaign/ReleaseNotes/3_0_2.md:16]:
```suggestion
- Displays the occurrence date of the last campaign incident. (Available from Cortex XSOAR 5.5.0).
```

File: Packs/Campaign/ReleaseNotes/3_0_2.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+- Get the campaign lower similarity incident IDs as option values for Multi Select field.
+
+##### PerformActionOnCampaignIncidents
+- Added support for campaign lower similarity incident fields.
+
+##### New: ShowCampaignSimilarityRange
+- Displays the similarity range between the incidents that make up the phishing campaign. (Available from Cortex XSOAR 5.5.0).
+
+##### New: ShowCampaignRecipients
+- Displays the phishing campaign recipients email addreses and their amount. (Available from Cortex XSOAR 5.5.0).
```
Comment (dantavori) [inline Packs/Campaign/ReleaseNotes/3_0_2.md:13]:
```suggestion
- Displays the phishing campaign recipients email addresses and their amount. (Available from Cortex XSOAR 5.5.0).
```

File: Packs/Campaign/ReleaseNotes/3_0_2.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+
+##### New: ShowCampaignLastIncidentOccurred
+- Displays the date of the last campaign incident occurred. (Available from Cortex XSOAR 5.5.0).
+
+##### New: ShowCampaignSenders
+- Displays the phishing campaign senders email addreses and their amount. (Available from Cortex XSOAR 5.5.0).
```
Comment (dantavori) [inline Packs/Campaign/ReleaseNotes/3_0_2.md:19]:
```suggestion
- Displays the phishing campaign senders email addresses and their amount. (Available from Cortex XSOAR 5.5.0).
```

**Review by dantavori — CHANGES_REQUESTED** [review]
looks great! most of my notes are docs notes.

**Comment by dantavori** [comment]
P.S: please add readmes to all of the scripts

**Review by dantavori — APPROVED** [review]

**Comment by tschanfeld** [comment]
minor edits

**Pull request merged**